addons/search-profile/userscript.js

```diff
@@ -6,11 +6,7 @@ export default async function ({ addon, global, console, msg }) {
     img = li.appendChild(document.createElement("img")),
     span = li.appendChild(document.createElement("span")),
     user = document.querySelector('[name="q"]').value.trim(),
-    valid =
-      user.length <= 20 &&
-      user.length >= 3 &&
-      !user.includes(" ") &&
-      user.match(/([\x30-\x39]|[\x41-\x5a]|[\x61-\x7a]|\x2d|\x5f)+/g)[0] == user;
+    valid = user.test(/^[\w-]{3,20}$/g);
```

Comment:
Uncaught (in promise) TypeError: user.test is not a function

Reply:
whoops